src/components/Target.vue

```diff
@@ -2,8 +2,8 @@
   <p>
     {{ this.val }}
     {{ this.buslist !== "hello" ? hours + "시" : "" }}
```

Comment:
How about add "v-if" statement to show hours only at midnight?
I think without "hour" is more clear and readable.

Reply:
I think that is a good idea.